ecosystem/platform/server/docker-bake.hcl

```diff
@@ -4,11 +4,10 @@
 
 
 variable "GIT_SHA1" {}
-variable "AWS_ECR_ACCOUNT_URL" {}
+variable "AWS_ECR_ACCOUNT_NUM" {}
```

Comment:
Think you'll have to set this in CircleCI

Reply:
it's set in circleci